changelog/unreleased/change-add-multiple-selection-sidebar

```diff
@@ -0,0 +1,7 @@
+Change: Add multiple selection Sidebar
```

Comment:
@paulcod3 that's not a breaking change since it doesn't require interaction from an admin. Please rename the changelog item (file + heading) as `enhancement` :)